python/py-pdfminer/Portfile

```diff
@@ -2,9 +2,13 @@
 
 PortSystem          1.0
 PortGroup           python 1.0
-
+PortGroup           github 1.0
+github.setup        euske pdfminer 423f851
+version             20200118
```

Comment:
I think putting dashes in there makes it clearer that this is a date. Also that's how most other ports handle it. 

`version             2020-01-18`

Reply:
By a quick grep, 20 ports use YYYY-MM-DD, while 210 use YYYYMMDD.

Add to this the fact that this port currently uses YYYYMMDD, I plan to stick with that, unless there is some other compelling reason to change.

Reply:
Sounds good.